admin_guide/multiproject_quota.adoc

```diff
@@ -75,12 +74,45 @@ status:
 <3> A label selector that can be used to select projects.
 <4> A per-namespace map that describes current quota usage in each selected project.
 <5> The aggregate usage across all selected projects.
-====
 
 This multi-project quota document controls all projects requested by
 *<user-name>* using the default project request endpoint. You are limited to 10
 pods and 20 secrets.
 
+Similarly, to select projects based on labels, run the command:
+
+[source, bash]
+----
+$  oc create clusterresourcequota for-name \ <1>
+    --project-label-selector=name=frontend \ <2>
+    --hard=pods=10 --hard=secrets=20
+----
+
+<1> Both `clusterresourcequota` and `clusterquota` are aliases of the same
+command. `for-name` is the name of the `clusterresourcequota` object.
+<2> A label selector to select projects in the format
```

Comment:
Maybe "To select projects by label, provide a key-value pair using the format `--project-label-selector=key=value`."